File: two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, num: List[int], target: int) -> List[int]:
        start , end = 0 , len(num)-1
        while start < end:
            if num[start]+num[end]==target:
                return (start+1,end+1)
            if num[start]+num[end]>target:
                end -= 1
            else:
                start += 1

File: two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int>res ;
        int l = 0 , r = nums.size()-1;
        while(l<r){
            if(nums[l]+nums[r]==target){
                res.push_back(l+1);
                res.push_back(r+1);
            }
            if(nums[l]+nums[r]>target){
                r -= 1;
            }
            else{
                l ++;
            }
        }
        return res;
    }
};